### PR TITLE
Fix invalid YAML

### PR DESCRIPTION
**Pull request by jboxman**
Fix invalid YAML that cannot be parsed
    
The YAML in these files cannot be parsed successfully for several reasons:
    
- Incorrect indentation
- Incorrect [source,yaml] block when content is not YAML
- Missing --- for listing with multiple YAML blocks

### CODE REVIEW

File: modules/installation-disk-partitioning-upi-templates.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ spec:
             - label: var
               sizeMiB: 240000
               startMiB: 0
-            filesystems:
+              filesystems:
```
[the remark this answers is not in the document]
Comment (liangxia) [reply modules/installation-disk-partitioning-upi-templates.adoc:64]: @jianlinliu @gpei  Could you help check ? Thanks.

File: modules/builds-identifying-image-change-triggers.adoc [hunk 2]
```diff
@@ -12,38 +12,38 @@ As a developer, if you have image change triggers, you can identify which image
...
+      name: input:latest
+      namespace: bc-ict-example-namespace
+    lastTriggerTime: "2021-06-30T13:47:53Z"
+    lastTriggeredImageID: image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input@sha256:0f88ffbeb9d25525720bfa3524cb1bf0908b7f791057cf1acfae917b11266a69
+  - from:
+      name: input2:latest
+      namespace: bc-ict-example-namespace
+    lastTriggeredImageID:  image-registry.openshift-image-registry.svc:5000/bc-ict-example-namespace/input2@sha256:0f88ffbeb9d25525720bfa3524cb2ce0908b7f791057cf1acfae917b11266a69
+
+  lastVersion: 1
```
Comment (liangxia) [inline modules/builds-identifying-image-change-triggers.adoc:46]: /cc @xiuwang

File: modules/installation-disk-partitioning-upi-templates.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (jianlinliu) [reply modules/installation-disk-partitioning-upi-templates.adoc:64]: The change is not correct. The correct one should be like:
```
spec:
  config:
    ignition:
      version: 3.2.0
    storage:
      disks:
        - device: /dev/sdb
          partitions:
            - label: var
              sizeMiB: 240000
              startMiB: 0
      filesystems:
        - device: /dev/disk/by-partlabel/var
          format: xfs
          path: /var
    systemd:
      units:
        - contents: |
            [Unit]
            Before=local-fs.target
            [Mount]
            Where=/var
            What=/dev/disk/by-partlabel/var
            Options=defaults,pquota
            [Install]
            WantedBy=local-fs.target
          enabled: true
          name: var.mount
```

File: modules/installation-disk-partitioning.adoc [hunk 3]
```diff
@@ -69,28 +69,28 @@ spec:
...
-            filesystems:
-        - device: /dev/disk/by-partlabel/var
-          format: xfs
-          path: /var
+      - device: /dev/sdb
+        partitions:
+        - label: var
+          sizeMiB: 240000
+          startMiB: 0
+          filesystems:
```
Comment (gpei) [inline modules/installation-disk-partitioning.adoc:77]: The same as above, expected format should be:
```
spec:
  config:
    ignition:
      version: 3.2.0
    storage:
      disks:
        - device: /dev/sdb
          partitions:
            - label: var
              sizeMiB: 240000
              startMiB: 0
      filesystems:
        - device: /dev/disk/by-partlabel/var
          format: xfs
          path: /var
    systemd:
      units:
        - contents: |
            [Unit]
            Before=local-fs.target
            [Mount]
            Where=/var
            What=/dev/disk/by-partlabel/var
            Options=defaults,pquota
            [Install]
            WantedBy=local-fs.target
          enabled: true
          name: var.mount
```

File: modules/cluster-logging-deploying-about.adoc [hunk 4]
```diff
@@ -59,7 +59,7 @@ spec:
         requests:
           cpu:
           memory:
-     type: kibana
+      type: kibana
```
[the remark this answers is not in the document]
Comment (anpingli) [reply modules/cluster-logging-deploying-about.adoc:62]: @jboxman  We should not remove the prefix space.

File: modules/cluster-logging-elasticsearch-storage.adoc [hunk 5]
```diff
@@ -29,17 +29,15 @@ apiVersion: "logging.openshift.io/v1"
...
-          size: "200G"
+# ...
+spec:
+  logStore:
+    type: "elasticsearch"
+    elasticsearch:
+      nodeCount: 3
+      storage:
+        storageClassName: "gp2"
+        size: "200G"
```
[the remark this answers is not in the document]
Comment (anpingli) [reply modules/cluster-logging-elasticsearch-storage.adoc:40]: @jboxman  We shouldn't remove the prefix space.

File: modules/builds-identifying-image-change-triggers.adoc (hunk 2 above)
Comment (xiuwang) [reply modules/builds-identifying-image-change-triggers.adoc:46]: /cc @jitendar-singh  FYI, please check this change

File: modules/how-to-plan-your-environment-according-to-application-requirements.adoc [hunk 6]
```diff
@@ -82,90 +82,90 @@ links in the deployment's service specification file to overcome this:
...
-      creationTimestamp:
-      annotations:
-        description: This template will create a deploymentConfig with 1 replica, 4 env vars and a service.
-        tags: ''
-    objects:
-    - kind: DeploymentConfig
-      apiVersion: v1
+---
+Kind: Template
+apiVersion: v1
```
Comment (xiuwang) [inline modules/how-to-plan-your-environment-according-to-application-requirements.adoc:87]: The rows of this file should be alphabetized.

File: modules/how-to-plan-your-environment-according-to-application-requirements.adoc [hunk 7]
```diff
@@ -82,90 +82,90 @@ links in the deployment's service specification file to overcome this:
...
-      name: deploymentConfigTemplate
-      creationTimestamp:
-      annotations:
-        description: This template will create a deploymentConfig with 1 replica, 4 env vars and a service.
-        tags: ''
-    objects:
-    - kind: DeploymentConfig
-      apiVersion: v1
+---
+Kind: Template
```
Comment (xiuwang) [inline modules/how-to-plan-your-environment-according-to-application-requirements.adoc:86]: s/Kind/kind

File: modules/how-to-plan-your-environment-according-to-application-requirements.adoc [hunk 8]
```diff
@@ -82,90 +82,90 @@ links in the deployment's service specification file to overcome this:
...
+apiVersion: v1
+metadata:
+  name: deploymentConfigTemplate
```
Comment (xiuwang) [inline modules/how-to-plan-your-environment-according-to-application-requirements.adoc:89]: The Template "deploymentConfigTemplate" is invalid: metadata.name: Invalid value: "deploymentConfigTemplate": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

**Pull request merged**